mods/dhssb/random-teams.js

```diff
@@ -0,0 +1,249 @@
+'use strict';
```

Comment:
Wow cool I wanted to do this a long time ago

Reply:
Well I did it for you